includes/Modules/Analytics.php

```diff
@@ -521,7 +521,7 @@ protected function handle_provisioning_callback() {
 
 		// Retrieve the internal web property id.
 		try {
-			$web_property = $this->get_service( 'analytics' )->management_webproperties->get( $account_id, $property_id );
+			$web_property = $this->get_service( 'analytics' )->management_webproperties->get( $account_id, $web_property_id );
```

Comment:
fixes a php fatal: the variable was previously renamed and this instance missed.